tui-calendar 추가시 js 로드 부분 개선

skandla2002 opened the pull request:
- update: tui.calendar의 이슈중 spring boot에서 사용시 적용 방안에 대한 부분 참조하여 수정(jquery 미적용 버전으로 tui-dom.js lib 사용)
- update: tui.calendar의 가이드상 css3개와 tui-code-snippet.js 는 header 영역에 존재하도록 배치 변경

skandla2002 commented:
js 로드 부분을 일부 수정하였습니다.
확인 후 머지 승인 부탁드립니다.

griffinna reviewed (approved):
jquery 미존재 버전을 사용하신 이유가 있을까요?

griffinna commented:
[AWS](http://ec2-52-78-213-26.ap-northeast-2.compute.amazonaws.com:8080) 
위 링크에서 확인하실 수 있습니다.
Today 와 2021.01 사이에 이동 버튼이 보이지 않는데 .map 에서 발생하는 에러 때문일까요?

[Error] Failed to load resource: the server responded with a status of 404 () (tui-calendar.js.map, line 0)
[Error] Failed to load resource: the server responded with a status of 404 () (main.css.map, line 0)

skandla2002 committed:
tui-calendar 추가시 js 로드 부분 개선
- update: tui.calendar의 이슈중 spring boot에서 사용시 적용 방안에 대한 부분 참조하여 수정(jquery 미적용 버전으로 tui-dom.js lib 사용)
- update: tui.calendar의 가이드상 css3개와 tui-code-snippet.js 는 header 영역에 존재하도록 배치 변경

skandla2002 commented:
기존 적용하신 js들이 jquery 미존재일때 사용 하는 버전으로  적용하신 상태여서 jquery 없이 사용하는 버전으로 tui-dom.js를 추가하였습니다.

그리고 map 로드시 에러는 product 버전 js를 로드할때 개발자 도구를 열면 발생하는 현상이므로, 오류가 아닙니다.
참고 부탁드립니다.

griffinna commented:
@skandla2002 오호.. 네 감사합니다. 
혹시 tui-calendar 외에 사용하기에 편리한 (?) calendar 가 있다면 추천해주시면 감사하겠습니다 ㅎㅎ 
달력뷰에서 구현하고자하는 기능은 
1.  날짜를 하루 클릭해서 레이어를 띄우고 데이터 입력 후 insert
2. 입력된 데이터를 해당 날짜에 뱃지? 처럼 표시
3. 입력된 데이터 클릭해서 상세조회.... 
입니다.  ㅋㅋㅋ

아! 그리고 jquery 는 **footer.mustache** 에 추가해두긴했었습니당!
